test/contrib/test_reparam.py

```diff
@@ -49,7 +49,6 @@ def model():
 
     with handlers.trace() as tr:
         value = handlers.seed(model, 0)()
-    assert isinstance(tr["x"]["fn"], dist.TransformedDistribution)
```

Comment:
@fehiepsi - The `TransformedDistribution` will be the base distribution now, so I am not sure if this PR can go through until we resolve #610.

Reply:
Yeah, I will address it.